lib/Sema/TypeCheckStorage.cpp

```diff
@@ -2311,6 +2311,9 @@ IsAccessorTransparentRequest::evaluate(Evaluator &evaluator,
           break;
         }
       }
+      if (auto subscript = dyn_cast<SubscriptDecl>(storage)) {
```

Comment:
Subscript setters just haven't been synthesized before. This prevents us from falling into `llvm_unreachable` a couple lines below. Since we're not worried about property wrappers for C++ interop, `SubscriptDecl`s don't need any special treatment that `VarDecl`s get above.
I kinda think that the comment right below this line applies here as well, but let me know if you'd like to have another comment here :)

Reply:
No, that's fine. Thanks.